squidlib-util/src/main/java/squidpony/panel/ICombinedPanel.java

```diff
@@ -107,12 +107,15 @@
 	 * {@code fgc} for the foreground.
 	 */
 	void put(int x, int y, String s, T bgc, T fgc);
-	
+
 	/**
+	 * @param what
+	 *            {@code true} to fill the foreground, {@code false} to fill the
+	 *            background. {@code null} to fill both.
 	 * @param color
 	 *            The color to put within this panel.
 	 */
-	void fillBG(T color);
+	void fill(Boolean what, T color);
```

Comment:
Personally, I'd prefer an enum rather than a boxed Boolean, since it's a little hard to tell what true, false or null mean here. I think static final fields, like LOS.BRESENHAM, would work if this weren't an interface.